### PR TITLE
Issue #118 - Add cron bundle to project

### PR DESCRIPTION
**Pull request by Radcho**
Ok.. toto je zvlastne. Bude treba pridat popis do rozbehavania s tadeto: https://github.com/Cron/Symfony-Bundle#installation
Joby sa registruju cez cron:create a potom sa v crone vola cron:run

### CODE REVIEW

**Comment by ZoltanOnody** [comment]
napis prosim do readme ako sa to spusta, potom to mergneme.

**Comment by Radcho** [comment]
Review needed

File: README.md [hunk 1]
```diff
@@ -56,6 +56,23 @@ Database client version: libmysql - 10.1.26-MariaDB
 PHP version: 7.0
 ```
 
+A CRON bundle was installed to help run CRON tasks in the production enviroment. To create a cron job, run:
+```
+docker-compose exec php bash
+php bin/console cron:create
+```
+The following commands fro sending emails are available:
```
Comment (ZoltanOnody) [inline README.md]: fro -> for

File: README.md [hunk 2]
```diff
@@ -56,6 +56,23 @@ Database client version: libmysql - 10.1.26-MariaDB
...
+- `email:delete:obsolete-registrations` - Deletes expired unclaimed registrations from the database, emptying space for other attendees.
+- `email:send:attendee-notification` - Sends reminders to all attendees that an upcoming event is nearing. Should be executed at least once a day.
+- `email:send:organizer-notification` - Sends information about registered users to organizers. Should be executed once a day.
+- `email:send:registration-open` - Sends notifications that a new event has become available for registration. Should be executed at least once a day.
+
+Setting up CRON tasks has to be done manually, either using CRON externally or using the CRON package avialble here: https://github.com/Cron/Cron
```
Comment (ZoltanOnody) [inline README.md]: avialble -> available

File: README.md [hunk 3]
```diff
@@ -56,6 +56,23 @@ Database client version: libmysql - 10.1.26-MariaDB
...
+
+The commands **have** to be executed from the bin folder.
```
Comment (ZoltanOnody) [inline README.md]: **have** to -> **have to**

**Review by ZoltanOnody — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Radcho) [reply README.md]: Toto je odplata?

Comment (ZoltanOnody) [reply README.md]: nieco take :-D

**Review by ZoltanOnody — APPROVED** [review]

**Pull request merged**